qemu/tests/block_resize.py

```diff
@@ -67,14 +67,16 @@ def compare_block_size(session, block_cmd, block_pattern):
                                                      data_dir.get_data_dir())
     data_image_dev = vm.get_block({'file': data_image_filename})
 
-    drive_path = ""
-    if params.get("os_type") == 'linux':
+    if (params.get("vm_arch_name") in ('ppc64', 'ppc64le') and
+            params.get("os_type") == 'linux'):
```

Comment:
blk_extra_params_stg is introduced since f83f336.
If we remove it, the block_resize works well.